Editor/UI/Wizard/Setup/XRSDKSetupPage.cs

```diff
@@ -38,8 +38,8 @@ private enum XRLoader
             { XRLoader.OpenVR, "OpenVR XR Plugin will be imported into the project." },
             { XRLoader.Oculus, "Oculus XR Plugin will be imported into the project." },
             { XRLoader.WindowsMR, "Windows XR Plugin will be imported into the project." },
-            { XRLoader.Other, "Right now we do not support other than the listed plugins." },
-            { XRLoader.None, "If you dont want to import any XR related plugins, press the skip button." }
+            { XRLoader.Other, "Right now we only support the listed plugins above." },
```

Comment:
Technically we support any Unity XR plugin, we just facilitate setting up the ones listed.

Reply:
So how would you say it?

Reply:
https://github.com/Innoactive/Creator/blob/024c94d045f46086d1e21f7e5106b1514b383e81/Editor/UI/Wizard/FirstTime/XRSDKSetupPage.cs#L112